src/Auth0.AuthenticationApi/Models/AuthorizationCodeTokenRequest.cs

```diff
@@ -8,6 +8,6 @@ public class AuthorizationCodeTokenRequest : AuthorizationCodeRequestBase
         /// <summary>
         /// Client secret of the application.
         /// </summary>
-        public string ClientSecret { get; set; }
+        public override string ClientSecret { get; set; }
```

Comment:
Should add a comment explaining the override.

Reply:
I decided to just remove the virtual/override and have a single set of docs on the base client that indicates the optional/required is dependent on class.